### PR TITLE
Reintroduce feedback form

### LINKED ISSUE #838: Reintroduce feedback form
On the Discord Art suggested we simply reintroduce the feedbcak form on the help pages

### PR DESCRIPTION
**Pull request by sabakhilji**
Closes #838 
Reintroduced feedback form at the end of every resource page.

### CODE REVIEW

File: ui/common/ResourcePage.tsx [hunk 1]
```diff
@@ -46,6 +47,10 @@ export default function ResourcePage({
             <div className="flex flex-col gap-[15px]">{codeResources}</div>
           </>
         )}
+        <>
+        <hr className="my-7 h-[1px] w-full opacity-25" />
+        <Text className="text-3xl" >{t('help_page.feedback')} </Text>
```
Comment (benalleng) [inline ui/common/ResourcePage.tsx:52]: lets use the footer tag here and make the text quite a bit smaller,  think its too big

**Review by benalleng — CHANGES_REQUESTED** [review]

File: ui/common/ResourcePage.tsx (hunk 1 above)
Comment (sabakhilji) [reply ui/common/ResourcePage.tsx:52]: did you mean the footer component?

Comment (benalleng) [reply ui/common/ResourcePage.tsx]: no i think its fine like this

File: i18n/locales/en.ts [hunk 2]
```diff
@@ -1678,6 +1678,7 @@ const translations = {
     solution_eight: 'Solution to part eight of this challenge',
     solution_nine: 'Solution to part nine of this challenge',
     solution_ten: 'Solution to part ten of this challenge',
+    feedback:'Your feedback is valuable and help us in enhancing our work more, please give us a feedback using <a href="https://docs.google.com/forms/d/e/1FAIpQLSf1xpNqUYJyvYL5IZDnxy78273pkqzfYW2Hf91H4Do4KHgy9g/viewform" target="_blank" rel="noreferrer">this link</a>. ',
```
Comment (benalleng) [inline i18n/locales/en.ts:1681]: should be helps not help

**Review by benalleng — CHANGES_REQUESTED** [review]

File: i18n/locales/en.ts (hunk 2 above)
Comment (sabakhilji) [reply i18n/locales/en.ts]: Oh yeah, nice catch!

**Review by benalleng — APPROVED** [review]

**Pull request merged**